src/renderer/js/component/publishForm/index.js

```diff
@@ -1,5 +1,11 @@
 import React from "react";
 import { connect } from "react-redux";
 import PublishForm from "./view";
+import { selectBalance } from "redux/selectors/wallet";
+import { formatFullPrice } from "util/formatCredits";
 
-export default connect(null, null)(PublishForm);
+const select = state => ({
+  balance: formatFullPrice(selectBalance(state) || 0, 10),
```

Comment:
Why format this? Also we should probably trust selector to fallback to 0?

Reply:
Ignore this for now. Just copy/pasted from somewhere else